### PR TITLE
feat: add new capabilities 2api endpoint

### PR DESCRIPTION
**Pull request by JeremyDec**
Service related to new 2api endpoint : /vps/capabilities/{serviceName}

### CODE REVIEW

File: src/api/capabilities/capabilities-aapi.service.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+angular.module("ovh-api-services").service("OvhApiCapabilitiesAapi", function ($cacheFactory, $resource) {
+    "use strict";
+
+    var cache = $cacheFactory("OvhApiCapabilitiesAapi");
+
+    var capabilitiesAapi = $resource("/capabilities/:feature/:serviceName", {
+        feature: "@feature",
+        serviceName: "@serviceName"
+    }, {
+        get: {
```
Comment (marie-j) [inline src/api/capabilities/capabilities-aapi.service.js]: `query` if it's an Array, no ?

Comment (JeremyDec) [reply src/api/capabilities/capabilities-aapi.service.js]: hmm yes it's better

**Comment by JeremyDec** [comment]
@antleblanc It is done

**Comment by jleveugle** [comment]
Hello @JeremyDec,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request merged**